pkg/scheduler/core/generic_scheduler.go

```diff
@@ -188,3 +191,101 @@ func (g *genericScheduler) chooseSpreadGroup(spreadGroup *util.SpreadGroup) []st
 	}
 	return feasibleClusters
 }
+
+func (g *genericScheduler) assignReplicas(clusters []*clusterv1alpha1.Cluster, replicaSchedulingStrategy *policyv1alpha1.ReplicaSchedulingStrategy, object *workv1alpha1.ObjectReference) ([]workv1alpha1.TargetCluster, error) {
+	if len(clusters) == 0 {
+		return nil, fmt.Errorf("no clusters available to schedule")
+	}
+	if object.Replicas == 0 {
+		targetClusters := make([]workv1alpha1.TargetCluster, len(clusters))
+		for i, cluster := range clusters {
+			targetClusters[i] = workv1alpha1.TargetCluster{Name: cluster.Name}
+		}
+		return targetClusters, nil
+	}
+	if replicaSchedulingStrategy != nil && replicaSchedulingStrategy.WeightPreferences != nil {
+		return g.calculateReplicasWithWight(clusters, replicaSchedulingStrategy.WeightPreferences.StaticWeightList, object.Replicas)
```

Comment:
Maybe we need to discuss about the dafault schedule strategy. If default schedule strategy is aggregated, most of the e2e test cases will failed, because these cases are available in the duplicate scenes.

Reply:
I think so.  Shall we discuss it?

Reply:
We can book a meeting! cc @RainbowMango @GitHubxsy

Reply:
+1 how about today?